pytest.ini

```diff
@@ -1,10 +0,0 @@
-[pytest]
```

Comment:
whats up with this change?

Reply:
Woops, thanks, good catch. Think I deleted that locally trying to get some VScode stuff to work. Added back in